06week/higherOrder.js

```diff
@@ -3,15 +3,45 @@
 const assert = require('assert');
 
 function forEach(arr, callback) {
-  // Your code here
+  // create an array with whatever you want in it
+  const arrProblemOne = [1, 2, 3]
+
+  //make a for loop that runs the number of items in the array amount of times.
+  //make a variable that prints a phrase the amount of times defined by the for loop
+  for (let i = 0; i < arrProblemOne.length; i++) {
```

Comment:
Your for loop should loop around the arr that the user passes in, not a hard coded array.